src/main/java/org/spongepowered/api/command/CommandCallable.java

```diff
@@ -79,6 +79,20 @@
     boolean testPermission(CommandSource source);
 
     /**
+     * Test whether this command is allowed to be executed by the specified
```

Comment:
Take a look at javadocs in other parts of the API, you'll see we consistently use "Gets" "Sets" etc